game/static/game/css/opening_trainer.css

```
@@ -0,0 +1,65 @@
+body{
+    background:#111;
+    color:white;
+    font-family:Segoe UI,sans-serif;
```

Stylelint (17.13.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)